cmd/ferretdb/main_postgres.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build postgres
```

Comment:
Disabling  default initialization of the Postgres handler:
* compile conflict with `dummy` handler that's enabled always
* if tigris is enabled, we don't need a second backend

Reply:
Sorry, no, pg handler should always be built

Reply:
> Sorry, no, pg handler should always be built

ok, changed.